src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/modelchecking/ModelCheckingStrategy.kt

```diff
@@ -367,27 +308,32 @@ internal class ModelCheckingStrategy(
             threadSwitchChoices.add(iThread)
         }
 
-        fun addCrashPosition(crashPosition: Int, isSystemCrash: Boolean) {
-            crashPositions.add(crashPosition)
-            if (!isSystemCrash) nonSystemCrashes.add(crashPosition)
-        }
-
         fun addLastNoninitializedNode(lastNoninitializedNode: InterleavingTreeNode) {
             this.lastNoninitializedNode = lastNoninitializedNode
         }
-
-        fun build() = Interleaving(switchPositions, crashPositions, threadSwitchChoices, nonSystemCrashes, lastNoninitializedNode)
     }
+}
 
-    private enum class ExplorationNodeType {
-        SWITCH, CRASH, NONE;
-
-        companion object {
-            fun fromNode(node: InterleavingTreeNode?) = when (node) {
-                is SwitchChoosingNode -> SWITCH
-                is CrashChoosingNode -> CRASH
-                else -> NONE
-            }
-        }
+internal class ModelCheckingStrategy(
```

Comment:
Lets separate these classes (`ModelCheckingStrategy` and `AbstractModelCheckingStrategy`) into different files

Reply:
We will lose all Git Blame(If I name files ModelCheckingStrategy.kt and AbstractModelCheckingStrategy.kt respectively). Is it OK?

Reply:
I think it's alright, the git blame of the file is not very helpful